README.md

```diff
@@ -211,7 +211,7 @@ Nginx, MariaDB, ElasticSearch and Redis.
 4. Run migrations `bin/console doctrine:migrations:migrate`
 5. Create ES search index `bin/console fos:elastica:create`
 6. Run `vendor/bin/phpunit` and `vendor/bin/behat` to ensure your test suite is
-   working
+   working. Run `bin/console fos:elastica:create --env=test` to create the test index.
```

Comment:
This is not necessary. ES indexes are reset by `resetIndex()`in the `FeatureContext`: https://github.com/itk-dev/ddb-cover-service/blob/99575b4f962c31480372f3f1edee49ffae61f076/features/bootstrap/FeatureContext.php#L144